src/ui/edit-content/EditContentForm.js

```diff
@@ -275,7 +275,7 @@ export class EditContentFormBody extends React.Component {
                         optionValue="code"
                         optionDisplayName="name"
                         defaultOptionId="cms.label.chooseoption"
-                        disabled={workMode === WORK_MODE_EDIT ? true : ownerGroupDisabled}
+                        disabled={workMode === WORK_MODE_EDIT}
```

Comment:
`OwnerGroup` is influencing couple of attributes in the content ( I mean Content Attributes ).
That's why it was immutable in the beginning.
Since its mutable now, following Attributes should be reset/cleared/cleaned/emptied (whatever :D) when owner group changes. (Because, depending on owner group, user can choose corresponding values for those attributes).
The affected attributes are:

- AttachAttributeField
- HypertextAttributeField
- ImageAttributeField
- LinkAttributeField
- Potentially CompositeAttributeField too, as it may contain any attribute from this list (apart from Monolist)
- Potentially MonolistAttributeField too, as it also may contain any attribute from this list above (apart from Composite)

Reply:
got it, thanks for the heads up!

Reply:
@ichalagashvili I opt to use `resetSection` to clear all values from `attributes` section when changing the owner group, do you think it is good enough?

Reply:
I am not sure if UX approves it or not ( they may say that its inconvenient if other non-affected attributes will be cleared out too, but lets see). Should I approve?

Reply:
I'll ask it in JIRA, lez hold it until we get an answer.